lib/fluent/plugin/output.rb

```diff
@@ -326,6 +326,7 @@ def start
           @buffering = prefer_buffered_processing
           if !@buffering && @buffer
             @buffer.terminate # it's not started, so terminate will be enough
+            @buffer = nil
```

Comment:
No. `@buffering` shows this plugin is buffered or non-buffered. Unassigning `@buffer` is just to avoid side-effect in `in_monitor_agent`.